eng/native/tryrun.cmake

```diff
@@ -145,8 +147,12 @@ elseif(TARGET_ARCH_NAME MATCHES "^(armel|arm|armv6|arm64|loongarch64|riscv64|s39
     set_cache_value(HAVE_FULLY_FEATURED_PTHREAD_MUTEXES 1)
     set_cache_value(HAVE_FUNCTIONAL_PTHREAD_ROBUST_MUTEXES_EXITCODE 0)
   endif()
+
+  if (TIZEN)
+    set_cache_value(HAVE_FUNCTIONAL_PTHREAD_ROBUST_MUTEXES_EXITCODE 0)
+  endif()
 else()
-  message(FATAL_ERROR "Arch is ${TARGET_ARCH_NAME}. Only armel, arm, armv6, arm64, loongarch64, s390x, ppc64le and x86 are supported!")
+  message(FATAL_ERROR "Arch is ${TARGET_ARCH_NAME}. Only armel, arm, armv6, arm64, loongarch64, riscv64, s390x, ppc64le and x86 are supported!")
```

Comment:
Is it intentional to add riscv64 here? I believe that riscv64 is very incomplete, it is misleading to call it supported.

Reply:
I thought that it was missed in this comment, because it won't throw any errors and riscv64 is supported in condition above

Reply:
I've removed riscv64 change.

Reply:
Yes, this error message is not a statement about quality of support, but it only reflects what's in the `if` condition (which has riscv64). Previously we reworded these error messages which are growing too long. e.g. `Only armel, arm, armv6, arm64, loongarch64, s390x, ppc64le and x86 are supported!` can be removed.